packages/application-config/src/validations.ts

```diff
@@ -70,3 +70,17 @@ export const validateSubmenuLinks = (
     uriPathSet.add(uriPath);
   });
 };
+
+export const validateAdditionalOAuthScopes = (
+  config: JSONSchemaForCustomApplicationConfigurationFiles
+) => {
+  const additionalPermissionNames = new Set();
+  config.additionalOAuthScopes?.forEach(({ name }) => {
+    if (additionalPermissionNames.has(name)) {
```

Comment:
We should also validate that the name matches the entry point regex.

Reply:
Updated [5a7680f](https://github.com/commercetools/merchant-center-application-kit/pull/2822/commits/5a7680f81bd8b3aa964ec4be22e6f7341f825ec9)